lib/video/postprocessing/finetune/getTimingMetrics.js

```diff
@@ -52,6 +65,28 @@ export function getTimingMetrics(videoMetrics, timingMetrics, options) {
         value: vm.LastVisualChange
       }
     );
+
+    if (vm.KeyColorFrames) {
+      let keyColorFrames = jsonifyKeyColorFrames(vm.KeyColorFrames);
+      for (let key in keyColorFrames) {
+        for (let frame of keyColorFrames[key]) {
+          let start = frame.startTimestamp;
+          let end = frame.endTimestamp;
+          if (end > start) {
```

Comment:
You can fix probably fix most of those errors by running `npm run lint:fix` and then push the changes.